irispy/iris_tools.py

```diff
@@ -171,32 +174,39 @@ def get_iris_response(pre_launch=False, response_file=None, response_version=Non
             iris_response["DATE_OBS"] = parse_time(iris_response["DATE_OBS"])
         except:
             iris_response["DATE_OBS"] = None
-        # Convert C_F_TIME to array of datetime objects while
-        # conserving shape.
-        c_f_time = np.empty(iris_response["C_F_TIME"].shape, dtype=object)
-        for i, row in enumerate(iris_response["C_F_TIME"]):
-            for j, t in enumerate(row):
-                c_f_time[i][j] = parse_time(float(t))
-        iris_response["C_F_TIME"] = c_f_time
+
+        # # Convert C_F_TIME to array of datetime objects while
+        # # conserving shape.
+        # c_f_time = np.empty(iris_response["C_F_TIME"].shape, dtype=object)
+        # for i, row in enumerate(iris_response["C_F_TIME"]):
+        #     for j, t in enumerate(row):
+        #         c_f_time[i][j] = parse_time(float(t))
+        # iris_response["C_F_TIME"] = c_f_time
+        c_f_time = iris_response["C_F_TIME"]
```

Comment:
Delete this line and uncomment the above paragraph.

Reply:
Done.